bcml/mergetext.py

```diff
@@ -113,11 +114,18 @@ def main(path : Path, lang = 'USen'):
                     if mod_text['entries'] == ref_text['entries']: continue
                     for entry in mod_text['entries']:
                         if are_entries_diff(entry, ref_text, mod_text):
+                            diff_found = True
                             merged_lines['entries'][entry] = copy.deepcopy(mod_text['entries'][entry])
+                if diff_found:
+                    merge_count += 1
         merged_path = merged_dir / rel_path
         merged_path.parent.mkdir(parents=True, exist_ok=True)
         with open(merged_path, 'w', encoding='utf-8') as f_merged:
             yaml.dump(merged_lines, f_merged)
+        if merge_count > 0:
+            print(f'Merging {merge_count} versions of {rel_path}...')
+        else:
+            print(f'No merges necessary in {rel_path}...')
```

Comment:
It might be better not to log files that don’t need merging. Otherwise we’re inevitably listing text files that aren’t modified by even one mod. Could be misleading.